core/python_utils_test.py

```diff
@@ -61,11 +60,6 @@ def test_divide(self):
         self.assertEqual(python_utils.divide(4, 2), 2)
         self.assertEqual(python_utils.divide(5, 2), 2)
 
-    def test_url_unsplit(self):
-        response = urllib.parse.urlsplit('http://www.google.com')
-        self.assertEqual(
-            python_utils.url_unsplit(response), 'http://www.google.com')
```

Comment:
I don't see the "removal of python_utils.url_unsplit function" changes in this PR, is this expected?

Reply:
Ah, you're right Sandeep I missed this. @anurag629 Please remove `url_unsplit` from _python_utils.py_

Reply:
@vojtechjelinek  shoould i change it like this:
![Screenshot from 2022-01-13 06-57-28](https://user-images.githubusercontent.com/78868769/149249355-8179f8f9-5109-4292-b5aa-a1f515903ce4.png)